source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/calculation/CalculationSetOptions.java

```diff
@@ -40,6 +40,10 @@
   private boolean forceAggregation;
   private CalculationRoadOPS roadOPS = CalculationRoadOPS.DEFAULT;
   private int monitorSrm2Year;
+  /**
+   * If true apply the WNB 25km calculation limit. Default true for WNB.
+   */
+  private boolean wnbMaxDistance;
```

Comment:
Perhaps in this case `useWnbMaxDistance` makes more sense? With just `wnbMaxDistance` I would have expected an integer due to the `Distance` part.